File: 프로그래머스/0/120897. 약수 구하기/약수 구하기.cs
using System;
using System.Collections.Generic;

public class Solution 
{
    public int[] solution(int n) 
    {
        List<int> resultList = new List<int>();
        for(int i=1; i<=n; i++) {
            if(n%i == 0) {
                resultList.Add(i);
            }
        }
        return resultList.ToArray();
    }
}

File: 프로그래머스/0/120897. 약수 구하기/약수 구하기.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(int n) 
{
    vector<int> divisor;
    for(int i=1; i<=n ; i++)
    {
        if(n%i == 0)
        {
            divisor.push_back(i);
        }
    }
    return divisor;
}